runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -49,6 +49,7 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 private:
 #if defined(J9VM_GC_GENERATIONAL)
 	MM_GenerationalAccessBarrierComponent _generationalAccessBarrierComponent;	/**< Generational Component of Access Barrier */
+	MM_Scavenger *_scavenger;
 #endif /* J9VM_GC_GENERATIONAL */
 	MM_MarkingScheme *_markingScheme;
```

Comment:
please add annotations to new `_scavenger` as well as existed `_markingScheme` variables